### PR TITLE
Added kibana pod annotations

### PR DESCRIPTION
**Pull request by cliedeman**
- [x] Chart version *not* bumped (the versions are all bumped and released at the same time)
- [x] README.md updated with any new values or changes
- [x] Updated template tests in `${CHART}/tests/*.py` 
- [ ] Updated integration tests in `${CHART}/examples/*/test/goss.yaml`

### CODE REVIEW

File: kibana/tests/kibana_test.py [hunk 1]
```diff
@@ -333,3 +333,11 @@ def test_override_the_serverHost():
...
     assert c['env'][1]['name'] == 'SERVER_HOST'
     assert c['env'][1]['value'] == 'localhost'
+
+  def test_adding_pod_annotations():
+    config = '''
+podAnnotations:
+  iam.amazonaws.com/role: es-role
+'''
+    r = helm_template(config)
+    assert r['deployment'][uname]['spec']['containers'][0]['annotations']['iam.amazonaws.com/role'] == 'es-role'
```
Comment (Crazybus) [inline kibana/tests/kibana_test.py]: The key you are referring to here is for the annotations for the first container and not the entire pod. I think you want `assert r['deployment'][name]['spec']['template']['metadata']['annotations']['iam.amazonaws.com/role'] == 'es-role'`

File: kibana/tests/kibana_test.py [hunk 2]
```diff
@@ -333,3 +333,11 @@ def test_override_the_serverHost():
     c = r['deployment'][name]['spec']['template']['spec']['containers'][0]
     assert c['env'][1]['name'] == 'SERVER_HOST'
     assert c['env'][1]['value'] == 'localhost'
+
+  def test_adding_pod_annotations():
```
Comment (Crazybus) [inline kibana/tests/kibana_test.py]: There is some extra whitespace here which causes python to not run.

File: kibana/tests/kibana_test.py (hunk 1 above)
Comment (cliedeman) [reply kibana/tests/kibana_test.py]: That makes sense. Updated. I did this blind and was hoping jenkins would help me along 😄

File: kibana/tests/kibana_test.py (hunk 2 above)
Comment (cliedeman) [reply kibana/tests/kibana_test.py]: Fixed

**Comment by Crazybus** [comment]
jenkins test this please

**Pull request merged**